lib/codebreaker/statistics.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+module Codebreaker
+  class Statistics
+    include Constants::Shared
+
+    PERMITTED_CLASSES = [Codebreaker::User, Symbol].freeze
+    YAML_OPTIONS = { permitted_classes: PERMITTED_CLASSES, aliases: true }.freeze
+
+    def initialize(filename = 'default.yml')
+      @filename = filename
+      @data_store = []
```

Comment:
initialize reading from file here

Reply:
if I do it in initialization, I will read the file even when the user doesn't  want to see statistics, but  wants to write simply himself  in it. Maybe left reading file in  show? Need help